SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -418,8 +418,37 @@ theorem not_congr (e1 : a ≈ʷ b) : (~~~a) ≈ʷ ~~~b := by
   intros g h
   simp only [not_eq, e1 g h]
 
+theorem ofBitVec_not_eq_to : ofBitVec (~~~ x) ≈ʷ ~~~ ofBitVec x := by
+  intros _ a
+  simp [ofBitVec, a]
```

Comment:
```suggestion
theorem ofBitVec_not_eqTo : ofBitVec (~~~ x) ≈ʷ ~~~ ofBitVec x := by
  intros _ a
  simp [ofBitVec, a]
```
I guess `eq_to` is the name you've given `≈ʷ`? I would prefer it to be one camelCased component of the theorem name. I would parse the current name as `ofBitVec (~~~ x)` is equal to some rhs that is described by `to`